### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by tomchop**
Remove old files that we don't use anymore

### CODE REVIEW

File: ACKNOWLEDGEMENTS [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Acknowledgements: dftimewolf
 
-Copyright 2016 The dfTimeWolf Project Authors.
+Copyright 2023 The dfTimeWolf Project Authors.
```
Comment (joachimmetz) [inline ACKNOWLEDGEMENTS:3]: 2016-2023 you want to keep the original year

File: MANIFEST.in
Comment (joachimmetz) [inline MANIFEST.in:1]: not too familiar with poetry but isn't this still needed for an sdist?

**Review by joachimmetz — APPROVED** [review]
LGTM, some small comments

File: MANIFEST.in
Comment (tomchop) [reply MANIFEST.in:1]: Not that I remember. If it is, I'll bring it back.

File: ACKNOWLEDGEMENTS (hunk 1 above)
Comment (tomchop) [reply ACKNOWLEDGEMENTS:3]: Good to know, thanks!

**Pull request merged**